src/Compilers/CSharp/Test/Emit2/CodeGen/CodeGenSpanBasedStringConcatTests.cs

```diff
@@ -4,6 +4,7 @@
 
 #nullable disable
 
+using System.Linq;
 using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
```

Comment:
Is this using used?

Reply:
Assuming you meant the `Linq` using, yes, `Enumerable.Repeat` is used below.